spec/takeaway_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu, order, messenger) }
+  let(:menu) { double :menu, display_menu: {onion_bhaji: 2.95, chicken_tikka_masala: 7.95, naan: 1.95},has_dish?: true }
```

Comment:
Metrics/LineLength: Line is too long. [120/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)